Dockerfile

```
@@ -13,6 +13,11 @@ ENV REMOTE_SOURCE=${REMOTE_SOURCE:-.}
 
 WORKDIR /go/src/github.com/openshift/cluster-logging-operator
 
+# Copy and download Go dependencies first so they are cached before source changes.
+COPY ${REMOTE_SOURCE}/go.mod .
+COPY ${REMOTE_SOURCE}/go.sum .
```

Does it make sense to combine the two statements to have one layer containing both files?

```suggestion
COPY ${REMOTE_SOURCE}/go.mod ${REMOTE_SOURCE}/go.sum .
```